.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi**, and look through [issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=) before submitting.
+
+Check out our [wiki](https://github.com/alexa-pi/AlexaPi/wiki/Debugging), and [Audio Debugging](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging) in particular before filing an issue.
```

Comment:
Ah, sorry, I meant link to Debugging apart from the other links. So it should be like [wiki], [Debugging] and [Audio debugging].